buildbot/dependency.py

```diff
@@ -75,7 +75,7 @@ def do_dependency(args):
 
     # Checkout fixed version to avoid unexpected issues coming from upstream
     # Specific version can be uplifted as soon as such need arise
-    checkout_cmd = ["git", "checkout", "5f8249691ec8c25775789498951f8e9eb62c201d"]
+    checkout_cmd = ["git", "checkout", "9a3e962f16f5097d2054233ad8b6dad51b6f41b7"]
```

Comment:
The change looks good to me, but I thought that no one uses this script to pull icd loader sources. Usually CMake handles icd loader dependency (see opencl/CMakeLists.txt).

We probably should revisit this code separately.

Reply:
I just updated in both places to keep the code consistent :)